src/Xamarin.Android.Build.Tasks/Tasks/ManifestMerger.cs

```diff
@@ -32,9 +31,39 @@ public class ManifestMerger : JavaToolTask
 
 		public string ExtraArgs { get; set; }
 
+		/*
+		 * obj\Debug\AndroidManifest.xml:12:5-16:15 Error:
+		 * 	android:exported needs to be explicitly specified for element <service#crc642ecd25190884e03d.TestService>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.
+		 */
+		static readonly Regex manifestErrorRegEx = new Regex (@"(?<file>.+AndroidManifest\.xml):(?<line>\d+:\d+).+Error:(?<error>.+)?", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
will this also pick up standard Java errors that might be raised when running manifest merger?
There is a chance we could get both types of error from the tooling.

Reply:
Reviewing this again, this replaces this Regex:
https://github.com/xamarin/xamarin-android/blob/5f0a7da3d22c0968b39c73681ec0f459ebf3e311/src/Xamarin.Android.Build.Tasks/Tasks/JavaToolTask.cs#L69

Which (I think) shows errors in `*.java` files. We replace with the regex for `AndroidManifest.xml`.

This regex will also get used:
https://github.com/xamarin/xamarin-android/blob/5f0a7da3d22c0968b39c73681ec0f459ebf3e311/src/Xamarin.Android.Build.Tasks/Tasks/JavaToolTask.cs#L70

So we should still get messages if `manifestmerger` throws an Java exception, but I'm not sure how we test that?